docs/V1/Customer/get_respondents.md

```diff
@@ -48,37 +54,94 @@ POST https://api.scoringline.com/api-saas/v1/customer/questionnaires/foobar/resp
 
 Avec les paramètres, ci-dessous.
 
-Clé                        | valeur                              | description
----------------------------|-------------------------------------|---------------------------------
-filters[status]            | accepted                            | Par statut des répondants
-filters[customStatus]      | (integer-string-integer[]-string[]) | Passer les identifiants des statuts personnalisés
-filters[createdfrom]       | (integer) 1451602800                | Nouvelles candidatures depuis 
-filters[lastUpdatedFrom]   | (integer) 1466066108                | Candidatures mises à jour depuis
+Clé                        | Valeur       
+---------------------------|--------------
+filters[status]            | `accepted`   
+filters[createdfrom]       | `1451602800` 
+filters[lastUpdatedFrom]   | `1466066108` 
+
 
 Le script retournera un code `200` et l'ensemble des candidats ayant particpé au questionnaire à partir du 01/01/2016 00:00:00 et ayant été acceptés.
 
-Voici les données retournées
-
-data             | description
------------------|------------------
-id               | Id du répondant 
-score_auto       | Scoring automatique
-score_total      | Note total
-phone_number     | Numéro de téléphone du répondant
-main_comment     | Commentaire principal du répondant
-email            | Email du répondant 
-first_name       | Prénom du répondant 
-last_name        | Nom du répondant 
-candidate_link   | Lien vers la fiche du répondant (utilisateur connecté sur Scoringline) 
-synthesis_link   | Lien vers la synthèse PDF du répondant (utilisateur connecté sur Scoringline) 
-comments         | Tableau de commentaires concernant le répondant (voir tableau ci-dessous)
-custom_status    | Objet représentant le statut personnalisé attribué au répondant
-
-comments  | description
-----------|------------------
-author    | Prénom et Nom de l'auteur du commentaire
-author_id | Id de l'auteur du commentaire
-comment   | Commentaire
+Voici les données retournées:
+```json
+[
+  {
+    "score_total_percentage": null,
+    "id": 8865,
+    "score_auto": 37,
+    "score_total": null,
+    "main_comment": "Excellent candidat",
+    "phone_number": "+33666666666",
+    "email": "e.bachman@aviato.com",
+    "first_name": "Erlich",
+    "last_name": "Bachman",
+    "candidate_link": "https://fr.scoringline.com/admin/{yourCompanySlug}/questionnaire/foobar/repondant/8865/show",
```

Comment:
Maybe the questionnnaire/`foobar` should be replace by {yourQuestionnaireName} ? As you did for the company slug, is it too much ?